### PR TITLE
fix(storage): refine some definition

### PR DESCRIPTION
**Pull request by soulseen**

### CODE REVIEW

**Review by Ghostbaby — APPROVED** [review]

**Review by chlins — APPROVED** [review]
LGTM

File: apis/meta/v1alpha1/tls.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func (r ComponentWithTLS) GetName() string {
 type ComponentsTLSSpec struct {
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:Pattern="[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*"
-	CertificateRef string `json:"certificateRef"`
+	CertificateRef string `json:"certificateRef,omitempty"`
```
Comment (steven-zou) [inline apis/meta/v1alpha1/tls.go:31]: This field is a `Required` field, is `omitempty` suitable here?

**Review by steven-zou — COMMENTED** [review]
check comments

File: pkg/cluster/controllers/storage/provision.go [hunk 2]
```diff
@@ -88,7 +89,7 @@ func (m *MinIOController) getMinIOProperties(minioInstance *minio.Tenant) (*goha
 		endpoint = fmt.Sprintf("http://%s.%s.svc:%s", m.getServiceName(), m.HarborCluster.Namespace, "9000")
 	}
 
-	storageSpec.S3.RegionEndpoint = endpoint
+	storageSpec.S3.RegionEndpoint = strings.ToLower(endpoint)
```
Comment (heww) [inline pkg/cluster/controllers/storage/provision.go:92]: http://serverA and http://servera is the same thing?

Comment (soulseen) [reply pkg/cluster/controllers/storage/provision.go:92]: The DNS names are case insensitive

**Review by steven-zou — APPROVED** [review]
LGTM

**Pull request merged**